addons/headless/stringtable.xml

```diff
@@ -36,5 +36,11 @@
             <German>Zeichnet Transferstatistiken, Verbindungen und Verbindungsabbrüche in einer RPT-Datei auf. (Standard: Nein)</German>
             <Polish>Zapisz statystyki transferu i status połączenia Headless Clienta do RPT. (Domyślnie: Tak)</Polish>
         </Key>
+        <Key ID="STR_ACEX_Headless_BlacklistEden">
+            <English>Headless Blacklist</English>
+        </Key>
+        <Key ID="STR_ACEX_Headless_BlacklistEdenDesc">
+            <English>If blacklisted this group will not be transfered to headless clients.</English>
```

Comment:
"Disables transferring of this AI unit/group to Headless Clients. No effect on player units."